### PR TITLE
Add Amaral Lab Northwestern NICO 101 course

### PR DESCRIPTION
**Pull request by reeserich**
## What does this PR do?
**Add resource(s)** | Remove resource(s) | Add info | Improve repo

### Description
This is a free course which is also taught at Northwestern University by the Amaral Lab called NICO 101 - Introduction to Programming for Big Data.
### Why is this valuable (or not)?
This course gives an introduction to python with specificity to performing data science and covers topics in data analysis, image processing, web scraping, and natural language processing.
### How do we know it's really free?
The course is licensed under the MIT license. I am also a member of the Amaral Lab and can confirm that this material is free to use.
### For book lists, is it a book? For course lists, is it a course? etc.
This is a course in python for data science.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -360,6 +360,7 @@
 * [Introduction to Data Science in Python](https://www.coursera.org/learn/python-data-analysis) - Christopher Brooks (Coursera)
 * [MIT's Introduction to Computational Thinking and Data Science](https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-0002-introduction-to-computational-thinking-and-data-science-fall-2016/) - Eric Grimson, John Guttag, Ana Bell (MIT OpenCourseWare)
 * [More Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/moredataminingwithweka/)
+* [NICO 101 - Introduction to Programming for Big Data](https://github.com/amarallab/Amaral_Lab_Intro_to_Data_Science) (Northwestern University - Amaral Lab)
```
Comment (eshellman) [inline courses/free-courses-en.md:363]:
```suggestion
* [NICO 101 - Introduction to Programming for Big Data](https://github.com/amarallab/Amaral_Lab_Intro_to_Data_Science) - Luis Amaral, Helio Tejedor
```

Comment (reeserich) [reply courses/free-courses-en.md]: Updated.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -971,6 +972,7 @@
 * [Learn Python Tutorial](https://www.kaggle.com/learn/python) (Kaggle)
 * [Learn to program in Python](https://www.codecademy.com/learn/python)
 * [Learn to Program: The Fundamentals](https://www.coursera.org/learn/learn-to-program) (Coursera)
+* [NICO 101 - Introduction to Programming for Big Data](https://github.com/amarallab/Amaral_Lab_Intro_to_Data_Science) (Northwestern University - Amaral Lab)
```
[the remark this answers is not in the document]
Comment (reeserich) [reply courses/free-courses-en.md]: Updated.

**Comment by eshellman** [comment]
sorry for the delay. see #9109 
Thanks, and welcome to free-programming-books!

**Comment by reeserich** [comment]
> sorry for the delay. see #9109 Thanks, and welcome to free-programming-books!

Thank you!

**Pull request merged**